### DISCUSSION TITLE
Hydra+Kratos Integrations

### DISCUSSION THREAD

**Discussion by amakarov70** [ory/hydra#3364] [Q&A]
Hi all! Many thanks to all the authors for the work done in the project. For several days I have been trying to implement a demo of the integration of MyApp + Kratos + Kratos-selfservice-ui-node + Hydra. The whole chain assembly does not work. I can't find an full example with current versions in the repositories. Please tell me where it can be found. Compose or K8s. Thanks in advance.

**Comment by chandanjog** [comment]
Had the same issue but was able to get a spike to test out how all the components stitch together. You can check this out - https://github.com/chandanjog/ory-suite-spike/blob/hydra-integration-2021/contrib/hydra/docker-compose.yml. Hope it helps.
